app/assets/javascripts/discourse/app/components/header.gjs

```diff
@@ -222,7 +222,7 @@ export default class GlimmerHeader extends Component {
       </div>
       <PluginOutlet
         @name="after-header"
-        @outletArgs={{hash minimized=this.header.topic}}
+        @outletArgs={{hash minimized=(not (not this.header.topic))}}
```

Comment:
Since we're in gjs... I think this should work, right? 👀 
```suggestion
        @outletArgs={{hash minimized=(Boolean this.header.topic)}}
```

Reply:
![](https://media1.tenor.com/m/nZ9IFupOViUAAAAC/keanu-reeves.gif)